benchmarks/benches/room_bench.rs

```diff
@@ -186,6 +187,8 @@ pub fn load_pinned_events_benchmark(c: &mut Criterion) {
 
     group.bench_function(BenchmarkId::new("load_pinned_events", name), |b| {
         b.to_async(&runtime).iter(|| async {
+            client.event_cache().subscribe().unwrap();
```

Comment:
Is there really a need to do it for *each* iteration of the benchmark? I don't think so, it should be subscribed to once and for all.

Reply:
Maybe I can wrap `Client` in an `Arc` and use `runtime.spawn_blocking` to run that code? Is there a better solution?

Reply:
Tried fixing this in https://github.com/matrix-org/matrix-rust-sdk/pull/3840/commits/807c87525854af90989e9920f5d5ed07254ed60a, please let me know if there's a better solution.

Reply:
Excellent, thanks!